### DISCUSSION TITLE
Add latest tag on the latest image

### DISCUSSION THREAD

**Discussion by julienLemarie** [aquasecurity/trivy#5960] [Ideas]
### Description

Hello,
It would be very helpful to have a tag "latest" to use the last version of Trivy.
For example, the dedicated task on Azure DevOps (https://marketplace.visualstudio.com/items?itemName=AquaSecurityOfficial.trivy-official) is always using an old version of Trivy because it need a manual change in the code of the task.
Thank you.

### Target

Container Image

### Scanner

None

**Comment by knqyf263** [comment]
We have it.
https://github.com/aquasecurity/trivy/pkgs/container/trivy/165932940?tag=latest

**Reply by itaysk** [reply]
OP is referring to Azure Devops plugin, not to GHCR image. The Azure Devops plugin really has the latest version hardcoded, there's an open issue about this https://github.com/aquasecurity/trivy-azure-pipelines-task/issues/45 I'll we can get someone to fix that.

**Reply by knqyf263** [reply]
OK. I'll close it as it is not Trivy's issue.

**Reply by simar7** [reply]
Not to beat a dead horse but @itaysk isn't the latest version of the plugin using a more recent version of trivy? https://github.com/aquasecurity/trivy-azure-pipelines-task/blob/main/trivy-task/index.ts#L7
